lib/statusbar-summary-view.coffee

```diff
@@ -27,16 +27,24 @@ class StatusBarSummaryView
     @tile = statusBar.addRightTile({item: el, priority: 100})
 
 class StatusBarSummary extends View
+  initialize: ->
+    @on 'click', '.linter-summary-container', ->
+      console.log("moveToNextMessage")
```

Comment:
Sorry for being unclear, I updated #424 with a description.

It's fine to start off by moving that function to utils btw. The architecture can change later if someone is motivated. Having it in utils isn't that bad.